extensions/amp-story-interactive/0.1/amp-story-interactive-quiz.css

```diff
@@ -187,6 +190,12 @@
   line-height: 1.25em !important;
 }
 
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-quiz-option:focus::before {
+    left: 0px !important;
```

Comment:
This looks random?

Also, RTL support?

Reply:
Actually I can fix this on another place. Seems like if you don't set `left: 0px` on an absolute element, it lines up with the padding of the parent so it wasn't being aligned with the button. 

RTL support is the same since the width is 100%, it's the same to do `left: 0` or `right: 0`